WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -305,9 +305,23 @@ class MediaCoordinator: NSObject {
                             success: {
                                 self.end(media)
         }, failure: { error in
-            guard let nserror = error as NSError? else {
-                return
-            }
+            // Ideally the upload service should always return an error.  This may be easier to enforce
+            // if we update the service to Swift, but in the meanwhile I'm instantiating an unknown upload
```

Comment:
There's no way to enforce an error in the Obj-C code?

Reply:
But I didn't say that.  I just said it's easier to enforce in Swift because the nullability of types isn't optional.  You have to clearly define if something is optional or not, whereas in Obj-C by default you can just avoid specifying anything.